Patch to install the Fortran modules and the pkgconfig file.
author: Alin M. Elena (STFC UKRI)
--- CMakeLists.txt.orig	2020-02-24 16:34:41.401661825 +0000
+++ CMakeLists.txt	2020-02-24 18:55:41.086406296 +0000
@@ -52,3 +52,10 @@ ENDIF()
 CONFIGURE_FILE(${PROJECT_SOURCE_DIR}/.elsi_rci.pc.in
   ${CMAKE_ARCHIVE_OUTPUT_DIRECTORY}/pkgconfig/elsi_rci.pc
   @ONLY)
+install(
+  FILES   ${PROJECT_BINARY_DIR}/lib/pkgconfig/elsi_rci.pc
+  DESTINATION ${CMAKE_INSTALL_LIBDIR}/pkgconfig
+  COMPONENT pkgconfig)
+
+install(DIRECTORY ${CMAKE_Fortran_MODULE_DIRECTORY}/ DESTINATION include/
+        FILES_MATCHING PATTERN "*.mod")
